File: app/src/main/java/com/mifos/passcode/MifosApplication.java
package com.mifos.passcode;

import android.app.Application;

import com.mifos.mobile_passcode.utils.ForegroundChecker;

/**
 * Created by dilpreet on 19/01/18.
 */

public class MifosApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        //need to initialize this
        ForegroundChecker.init(this);
    }
}

File: app/src/main/java/com/mifos/passcode/MifosApplication.kt
package com.mifos.passcode

import android.app.Application
import com.mifos.mobile.passcode.utils.ForegroundChecker.Companion.init
import dagger.hilt.android.HiltAndroidApp

/**
 * Created by dilpreet on 19/01/18.
 */

@HiltAndroidApp
class MifosApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        //need to initialize this
        init(this)
    }
}